dymos/transcriptions/solve_ivp/components/segment_simulation_comp.py

```diff
@@ -255,6 +255,9 @@ def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
                         rtol=self.options['rtol'],
                         t_eval=t_eval)
 
+        if not sol.success:
+            raise RuntimeError('solve_ipv failed', sol.message)
```

Comment:
solve_ipv -> solve_ivp

Also, please change from RuntimeError to AnalysisError from OpenMDAO.api (it's more specific and in the future we might be able to catch it and terminate simulate gracefully.

Reply:
:heavy_check_mark: https://github.com/OpenMDAO/dymos/pull/343/commits/7a9a687bc152833601b55ebf267db9c928393937